tools/webpack.prod.config.babel.js

```diff
@@ -45,7 +45,9 @@ const prodConf = {
     }),
     new HTMLWebpackPlugin({
       title: 'ToReplaceByTitle',
+      __UI_OPTIONS: 'ToReplaceByVerdaccioUI',
```

Comment:
`__UI_OPTIONS` will replace the options below in future tickets. The current solution does not scale, so, I plan to pass a JS object instead.